test/test_cloud_record.py

```diff
@@ -56,7 +56,7 @@ def setUp(self):
 SuspendDuration: NULL
 WallDuration: NULL
 CpuDuration: NULL
-CpuCount: 1
+CpuCount: 1.0
```

Comment:
Could some of these tests be changed so that they have a value after the decimal point?

Also, some of the data should still arrive with no decimal in it (as most cloud data is like that.